src/Illuminate/Foundation/Bootstrap/HandleExceptions.php

```diff
@@ -79,12 +79,13 @@ public function handleError($level, $message, $file = '', $line = 0, $context =
     /**
      * Reports a deprecation to the "deprecations" logger.
      *
+     * @param  int  $level
      * @param  string  $message
      * @param  string  $file
      * @param  int  $line
      * @return void
      */
-    public function handleDeprecation($message, $file, $line)
+    public function handleDeprecation($level, $message, $file, $line)
```

Comment:
This is a major breaking change. I think we have to wait until Laravel 10 to do this.

Reply:
If otherwise the implementation looks fine to you, I'll change the signature back and just hardcode a `E_DEPRECATED` in `new ErrorException()`. I don't think it's used by ErrorException anyway.